test/quantization/test_quantize_script.py

```diff
@@ -2089,6 +2089,25 @@ def test_hardswish(self):
             FileCheck().check_not("aten::hardswish") \
                        .run(m.graph)
 
+    def test_elu(self):
+        class FunctionalELU(torch.nn.Module):
+            def __init__(self, inplace):
+                super(FunctionalELU, self).__init__()
+                self.inplace = inplace
+
+            def forward(self, input):
+                return torch.nn.functional.elu(input, inplace=self.inplace)
+
+        modules = [torch.nn.ELU(), torch.nn.ELU(inplace=True),
+                   FunctionalELU(True), FunctionalELU(False)]
```

Comment:
looks like this can be decomposed as [torch.nn.ELU, FunctionalELU] and [True, False]

Reply:
makes sense.  Seems like it would not matter that much since inlining it is simple - lmk if you feel strongly

Reply:
the pattern will be repeated, and you have tracing = [True, False] as well. I think it's better to use itertools.

Reply:
we can do
```python
options = itertools.product([torch.nn.ELU, FunctionalELU], [True, False], [True, False])
for M, inplace, tracing in options:
 ...
```

Reply:
no opposition to itertools, but that's not what I was targeting in my comment, it was inlining the inplace versus automating it.  I guess I don't feel that strongly about this issue either, but I'm also not sure that the benefit here is worth the cost.  This is only 2 LOC and is really easy to read.

Reply:
I thought about this again, it's OK to keep it this way since it's a coincidence that both of the modules has `inplace` actually

Reply:
np, the suggestion sg